### PR TITLE
[MIG][9.0]purchase_order_analytic_search

### PR DESCRIPTION
**Pull request by gmeficent**

### CODE REVIEW

File: purchase_order_analytic_search/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import models
```
Comment (LoisRForgeFlow) [inline purchase_order_analytic_search/__init__.py:5]: Extra line

File: purchase_order_analytic_search/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Analytic Search",
+    "version": "9.0.1.0",
```
Comment (LoisRForgeFlow) [inline purchase_order_analytic_search/__openerp__.py]: you missed the last zero. The zeros at the right are always important! :euro: :euro: :euro: :stuck_out_tongue_closed_eyes:

File: purchase_order_analytic_search/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "depends": ["analytic", "purchase"],
+    "description": """
+Search purchase orders by analytic account. New menu entry in Purchasing to
+list purchase order lines.
+""",
+    "data": [
+        "views/purchase_order_view.xml",
+    ],
+    'installable': True,
+    'active': False,
```
Comment (LoisRForgeFlow) [inline purchase_order_analytic_search/__openerp__.py]: You can always remove this tag.

File: purchase_order_analytic_search/views/purchase_order_view.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+#---------------------------------------------------------------------------------------------------------
+#   Add analytic account id to purchase orders
+#---------------------------------------------------------------------------------------------------------
```
Comment (LoisRForgeFlow) [inline purchase_order_analytic_search/views/purchase_order_view.xml:6]: I didn't know you can comment like this in xml, that's good :+1:

File: purchase_order_analytic_search/README.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+It also introduces a new menu entry in Purchasing to list purchase order lines.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Set 'Analytic Accounting' and 'Analytic Accounting for
```
Comment (LoisRForgeFlow) [inline purchase_order_analytic_search/README.rst]: I would say that this fits better in the configuration section, doesn't it? :thinking:

**Review by LoisRForgeFlow — COMMENTED** [review]
Minor comments

**Comment by gmeficent** [comment]
Thank Lois, actually I applied the changes on this pull https://github.com/OCA/purchase-workflow/pull/383
I'll update here.

File: purchase_order_analytic_search/README.rst (hunk 5 above)
Comment (gmeficent) [reply purchase_order_analytic_search/README.rst]: I though so too

**Pull request closed without merging**